core/manifest/ConfigStaticManifest.php

```diff
@@ -183,17 +183,15 @@ function getStatics() {
 	}
 
 	/**
-	 * Get the next token to process, incrementing the pointer
+	 * Get the next token to process, incrementing the pointer, skips any whitespace tokens
 	 *
-	 * @param bool $ignoreWhitespace - if true will skip any whitespace tokens & only return non-whitespace ones
 	 * @return null | mixed - Either the next token or null if there isn't one
 	 */
-	protected function next($ignoreWhitespace = true) {
```

Comment:
The only way to overload the SS_ConfigStaticManifest_Parser is to straight up replace it in the framework and I really really doubt that anyone is extending this and using it in personal code.

However, if you guys are blocking this pull due to that, we can just close this and I can just make a pull where I only make the "$var == (array)$var" optimization, as that's where most of the speed up comes from.